lib/output.js

```diff
@@ -67,6 +71,14 @@ module.exports = function (parameters, response) {
     threshold
   ));
 
+  if (parameters.or) {
+    console.log('\nHere are your optimized images: ', optimizedResoruceURL);
+  }
+
+  if (parameters.download) {
+    got.stream(optimizedResoruceURL).pipe(createWriteStream('optimized.zip'));
+  }
```

Comment:
I don't think you do. That wouldn't help. Again, nothing to do with `got`.

You need to convert [this function](https://github.com/addyosmani/psi/blob/96f9579ab5a0084ff20faab7048e1448a22154f5/lib/output.js#L59) into an async one and call the callback function when `got` is done. The reason it's 0kb is that the CLI calls `process.exit` before `got` has even gotten around to start downloading it.